ros_bridge/main.py

```diff
@@ -25,6 +25,9 @@
 from chat_track import ChatTrack
 from face_track import FaceTrack
 from sound_track import SoundTrack
+from room_brightness import RoomBrightness
+from saliency_track import SaliencyTrack
```

Comment:
in the future (next few months or half-year) we need to figure out how to break the dependency on HEAD.  we should be able to get a working model, without having to force a dependency on HEAD.  For starters, maybe some sort of `try: import .. except: ...` would work, but it would be better if we had the sensors in some github repo that did not include all of the rest of the crap on HEAD.  This is a hard problem to solve, though.